### PR TITLE
added  footer with link to the teachLA site

### LINKED ISSUE #18: Link to Teach LA site
Add a link somewhere on the page that links back to the teach la site. Looking at [other learning labs](https://teachla.uclaacm.com/dev#projects) may give you a better idea of what the issue is asking for. Creating a design on the Figma and having someone review it first may be the best idea instead of trying to develop it right away.

### PR DESCRIPTION
**Pull request by connord28**
I added the footer but I needed to shrink it by a factor of 2 so that it didn't cover the next button and level select menu.
Also sometimes the image in the footer isn't loading properly but I don't know why.
I also added the method of linking the teachla site that we were going to use before I knew about the footer but that can't be seen unless the footer is removed.

### CODE REVIEW

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
hmu when this is ready for review!

File: src/styles/footer.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+#footer {
+  height: 20px;
+  left: 696px;
+  margin-top: 0;
+  position: absolute;
+  top: 95.5%;
+  width: 302px;
+}
```
Comment (archishadatta) [inline src/styles/footer.scss]: We should avoid setting things in terms of pixels. For size such as width and height, try to use vw (or any other responsive unit). For absolute positioning, use percent or vw usually.

Try something like
```
bottom: 2vw;
left: 50%;
```

Comment (archishadatta) [reply src/styles/footer.scss]: Also @reginawang99 love the new footer but can we make it an svg? Preferably in white, black, and full color! Font sizes are often variable and leaving it as raw text can cause issues like the one Matt asked me to fix in the template tonight.

**Review by archishadatta — CHANGES_REQUESTED** [review]

**Review by archishadatta — APPROVED** [review]

File: src/styles/footer.scss (hunk 1 above)
Comment (reginawang3495) [reply src/styles/footer.scss]: you're the designer so you know best! we had an offline discussion regarding svgs + links + selectable text

**Review by nendow02 — APPROVED** [review]

File: src/assets/FooterTxt.svg:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (reginawang99) [inline src/assets/FooterTxt.svg:Zone.Identifier:1]: can we delete this?

Comment (reginawang3495) [inline src/assets/FooterTxt.svg:Zone.Identifier]: can we delete this?

File: src/assets/teach-la-logo-full.svg:Zone.Identifier [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (reginawang3495) [inline src/assets/teach-la-logo-full.svg:Zone.Identifier]: can we delete these zone identifiers?

> identifier. Zone identifier files are generated automatically by Internet Explorer and other programs when files are downloaded to a Windows computer. ... These files are used by Windows manage security settings for specific files. They are typically hidden and are not meant to be opened directly.

File: src/assets/teach-la-logo-full.svg [hunk 4]
```diff
@@ -0,0 +1 @@
... 267.922C1516.6 267.922 1488 238.82 1488 202.922C1488 167.023 1517.9 137.922 1553 137.922C1578.35 137.922 1601 153.107 1611.61 175.07Z" clip-rule="evenodd"/><rect width="36" height="182.888" x="1632" y="83" fill="#000"/><path fill="#000" d="M1719.19 196.47L1755.19 187.546V265.882H1719.19L1719.19 196.47Z"/><path fill="#000" d="M1702.88 137.573C1737.79 137.573 1755.19 161.982 1755.19 187.573L1719.19 198.237C1719.19 184.813 1711.44 173.237 1693.92 173.237C1676.1 173.237 1668 186.237 1668 198.237L1657.09 182.124C1651.08 169.759 1667.88 137.573 1702.88 137.573Z"/><path fill="#000" d="M1789 83.9214H1828V263.921H1789V83.9214Z"/><path fill="#000" d="M1789 228.921H1903V265.921H1789V228.921Z"/><path fill="#000" d="M1991.8 83.9215L2023 83.9214L2021.51 105.55L1960.88 265.921L1923 265.921L1991.8 83.9215Z"/><path fill="#000" d="M2030.88 83.9215L2000 83.9214L2001.47 105.55L2061.5 265.921L2099 265.921L2030.88 83.9215Z"/><path fill="#000" d="M1996.12 83.9215L2027 83.9214L2025.53 105.55L1965.5 265.921L1928 265.921L1996.12 83.9215Z"/><path fill="#000" d="M2035.88 83.9215L2005 83.9214L2073.12 265.921L2104 265.921L2035.88 83.9215Z"/><path fill="#000" d="M1959 205.921H2066V238.921H1959V205.921Z"/></svg>
```
Comment (reginawang3495) [inline src/assets/teach-la-logo-full.svg]: i dont think we are using this?

File: src/components/shared/footer.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import '../../styles/footer.scss';
+import FooterTxt from '../../assets/FooterTxt.svg';
+
+function Footer(): JSX.Element {
+  return (
+    <div id="footer">
+      <h3>
+        <a href="https://teachla.uclaacm.com" target="_blank" rel="noopener noreferrer"><img id="tla-logo" src={FooterTxt} alt="Made with ❤️ by acm.teachLA"/></a>
```
Comment (reginawang3495) [inline src/components/shared/footer.tsx]: @archishadatta im looking at the svg, and i think it's a little blurry? I think we should probably do text and svgs? (can we use `em` instead? I got some input from other people as well!!)

we can probably discuss this offline as well if you want!

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
i think the svg is a little blurry!! but besides that, it looks pretty good :)

**Comment by reginawang3495** [comment]
Let's hold off on this PR for a bit

took this offline with @archishadatta 
> about the svg/text i’m thinking the best option rn is use text for everything except the actual logo part (the little diamond)
>
> I’m talking to crystal on monday for other reasons so i was planning to ask her what the correct protocol would be then so that’s why there’s a delay

File: src/styles/footer.scss [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+#footer {
+  align-items: center;
+  bottom: 1.3vw;
+  display: flex;
+  font-family: 'Westwood Sans','Helvetica Neue', 'Helvetica', sans-serif;
+  height: 1.35vw;
```
Comment (archishadatta) [inline src/styles/footer.scss]: Can you take out the height and width? It's not necessary. Also, this branch isn't updated with the mobile design so I don't know how the footer will look on mobile. But just by looking at the code, it should be okay. If it's not I'll fix it in my issue for the week.

**Review by archishadatta — CHANGES_REQUESTED** [review]

**Comment by nendow02** [comment]
Merged main and it does look like there is some css issues when it goes to mobile, but as already mentioned that falls more under Archie’s issue for this week.

**Review by archishadatta — APPROVED** [review]

File: src/styles/app.scss [hunk 7]
```diff
@@ -12,10 +12,18 @@
     display: inherit;
     height: fit-content;
     min-height: 100vh;
-    padding-bottom: 50px;
+    // padding-bottom: 50px;
```
Comment (reginawang3495) [inline src/styles/app.scss]:
```suggestion
```

File: src/styles/app.scss [hunk 8]
```diff
@@ -12,10 +12,18 @@
...
+  }
+
+  #contents {
+    padding-bottom: 60px;
   }
```
Comment (reginawang3495) [inline src/styles/app.scss]: duplicate key name

File: src/components/shared/footer.tsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import '../../styles/footer.scss';
```
Comment (reginawang3495) [inline src/components/shared/footer.tsx:1]: component file names should usually be captilized

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
some small things!

**Review by reginawang3495 — APPROVED** [review]
after those nits, it looks good to me!

**Pull request merged**